src/config/config.py

```diff
@@ -0,0 +1,58 @@
+"""Module to handle configuration application configuration parameters."""
+import configparser
+import logging
+from os.path import expanduser, isfile
+from pathlib import Path
+
+logger = logging.getLogger(__name__)
+config_file = "config.ini"
+
+
+def _find_config_directory() -> str:
+    # TODO: Check if windows or linux, use enviroment variables
```

Comment:
For checking operating system in the future you can use `os.name`. 

this should work for Linux and Mac for finding the right folders, https://pypi.org/project/xdg/. Maybe also windows.

Reply:
Check also `sys.platform` for more fine grained info of os. https://docs.python.org/3/library/sys.html?highlight=sys#sys.platform

Reply:
Seems like there are plenty of options 😅
https://stackoverflow.com/a/11674977

Reply:
Awesome, thanks! :smile: